### PR TITLE
Multitenancy documentation

### PR DESCRIPTION
**Pull request by ksamoray**
Add general multitenancy guide

### CODE REVIEW

File: website/docs/guides/multitenancy.html.markdown [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+NSX-T Project objects can be created and referenced with the `nsxt_policy_project` [resource](../r/policy_project.html.markdown) and [data source](../d/policy_project.html.markdown).
+
+For example, an NSX-T multi-tenancy Project could be created as below:
+
+```hcl
+resource "nsxt_policy_project" "test" {
+  display_name        = "test"
+  description         = "Terraform provisioned Project"
+  _default            = true
```
Comment (annakhm) [inline website/docs/guides/multitenancy.html.markdown:20]: not sure what `default` is here?

File: website/docs/r/policy_project.html.markdown [hunk 2]
```diff
@@ -17,7 +17,6 @@ This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
 resource "nsxt_policy_project" "test" {
```
Comment (annakhm) [inline website/docs/r/policy_project.html.markdown:17]: The line above - project should not be applicable for GM and VMC as of now

File: website/docs/r/policy_project.html.markdown [hunk 3]
```diff
@@ -47,6 +45,7 @@ In addition to arguments listed above, the following attributes are exported:
 * `id` - ID of the resource.
 * `revision` - Indicates current revision number of the object as seen by NSX-T API server. This attribute can be useful for debugging.
 * `path` - The NSX path of the policy resource.
+* `_default` - The server will populate this field when returing the resource. Ignored on PUT and POST.true - the project is a default project. Default projects are non-editable, system create ones.
```
Comment (annakhm) [inline website/docs/r/policy_project.html.markdown:48]: I don't think it is present in the resource at all? I'm not sure its useful, but if you'd like to add it, I think it should be named `default` or `is_default` to follow convention (we have `is_default` in tz and security policy data sources)

Comment (ksamoray) [reply website/docs/r/policy_project.html.markdown:48]: I think that it's useless. Maybe it's better if I'll remove it...

Comment (ksamoray) [reply website/docs/r/policy_project.html.markdown:48]: But maybe in a separate PR as this one is regarding docs - I'll get rid of this in the resource as well.

Comment (annakhm) [reply website/docs/r/policy_project.html.markdown:48]: I don't see it in resource schema: https://github.com/vmware/terraform-provider-nsxt/blob/master/nsxt/resource_nsxt_policy_project.go#L27, am I missing something?

Comment (ksamoray) [reply website/docs/r/policy_project.html.markdown:48]: Cool so I removed it already :)
I'll just fix the doc then. Thanks!!

**Review by annakhm — APPROVED** [review]

**Pull request merged**